lib/counter.js

```diff
@@ -4,20 +4,53 @@
 'use strict';
 
 const util = require('util');
-const type = 'counter';
-const { hashObject, isObject, getLabels, removeLabels } = require('./util');
+const {
+	hashObject,
+	isObject,
+	getLabels,
+	removeLabels,
+	nowTimestamp,
+} = require('./util');
 const { validateLabel } = require('./validation');
 const { Metric } = require('./metric');
+const Exemplar = require('./exemplar');
 
 class Counter extends Metric {
+	constructor(config) {
+		super(config);
+		this.type = 'counter';
+		this.defaultLabels = {};
+		this.defaultValue = 1;
+		this.defaultExemplarLabelSet = {};
+		if (config.enableExemplars) {
+			this.enableExemplars = true;
+			this.inc = this.incWithExemplar;
+		} else {
+			this.inc = this.incWithoutExemplar;
+		}
+	}
+
+	enableExemplars(enable = false) {
```

Comment:
are these `enableExemplars` methods used? what's the use case for them?

Reply:
I removed them, they were not used and implementing a toggle for exemplars on existing metrics is problematic.